web/app/help/operations/compute-segment-conductance.asciidoc

```diff
@@ -0,0 +1,25 @@
+### Compute segment conductance
+
+Computes the conductance of each segment in a non-overlapping segmentation.
```

Comment:
Shouldn't we enforce the non-overlappingness of the segmentation somewhere in the code? Probably the frontend operation should check the edge bundle properties of the belongs to bundle.